modules/apps/frontend-js/frontend-js-metal-web/package.json

```diff
@@ -8,7 +8,6 @@
 		"metal-assertions": "2.16.8",
 		"metal-clipboard": "2.0.1",
 		"metal-component": "2.16.8",
-		"metal-debounce": "^2.0.2",
```

Comment:
See comment about this at https://github.com/liferay-frontend/liferay-portal/pull/675#pullrequestreview-565206328.

If anyone now imports `metal-debounce`, it will try to fetch it from `frontend-js-metal-web` which no longer includes it. Granted, we've made sure that's not the case, but could happen for others.

As I said, I would go with removing the metal import configuration from the preset to be on the "safer" side and then push for a complete removal of this module.

/cc @dsanz

Reply:
@jbalsas isn't the whole point that we are saying don't use `metal-debounce` now? Which is why we are removing references to it.

Reply:
> @jbalsas isn't the whole point that we are saying don't use metal-debounce now? Which is why we are removing references to it.

Yeah, kinda. The thing is there's a difference between saying "don't use it" and breaking code that might be using it. One thing is a recommendation and the other a hard imposition.

As I said, removing the preset would make anyone using `metal-debounce` do so at their own risk and at least leave them with a functioning application. I also think the chances of this happening are very slim since we still haven't made the preset or npm-scripts a sanctioned external tool, so we know we're the sole users. 

It could be problematic if backported though, so maybe scratch all that 😂